src/main/scala/firrtl/Utils.scala

```diff
@@ -89,21 +89,14 @@ object Utils extends LazyLogging {
   val BoolType = UIntType(IntWidth(1))
   val one  = UIntLiteral(BigInt(1),IntWidth(1))
   val zero = UIntLiteral(BigInt(0),IntWidth(1))
-  def uint (i:Int) : UIntLiteral = {
-     val num_bits = req_num_bits(i)
-     val w = IntWidth(scala.math.max(1,num_bits - 1))
-     UIntLiteral(BigInt(i),w)
+  def uint(i: Int) : UIntLiteral = {
```

Comment:
To fully fix this spacing it should be:
```scala
def uint(i: Int): UIntLiteral = {
```
Also, I think we should move this to an actual apply method on UIntLiteral companion object itself. That doesn't have to be done in this commit though.